src/devices/src/virtio/rng/device.rs

```diff
@@ -110,19 +114,23 @@ impl Entropy {
         let mut used_any = false;
         while let Some(desc) = self.queues[RNG_QUEUE].pop(mem) {
             debug!("entropy: guest request for {} bytes of entropy", desc.len);
+            METRICS.entropy.entropy_event_count.inc();
 
             let index = desc.index;
             let bytes = self.handle_one(mem, desc).unwrap_or_else(|err| {
                 error!("entropy: {err}");
+                METRICS.entropy.entropy_event_fails.inc();
                 0
             });
 
             match self.queues[RNG_QUEUE].add_used(mem, index, bytes) {
                 Ok(_) => {
                     used_any = true;
+                    METRICS.entropy.entropy_bytes.add(bytes as usize);
```

Comment:
To you see this metric as "total bytes of random data consumed from host" or "total number of random bytes offered to guest"? If its the former, then the metric should be incremented inside of `handle_one` due to the fallible nature of that method.

Reply:
see previous comment.

Reply:
"number of bytes offered to guest" it is 😆